teuthology/task/kernel.py

```diff
@@ -992,6 +1013,11 @@ def task(ctx, config):
         kojihub_url: http://koji.fedoraproject.org/kojihub
         kojiroot_url: http://kojipkgs.fedoraproject.org/packages
 
+    When installing from a koji task_id you also need to set koji_task_url,
+    which is the base url used to download rpms from koji task results::
+
+        koji_task_url: https://kojipkgs.fedoraproject.org/work/
```

Comment:
I imagine you looked into this, but: is there really no way to figure this out based on ``kojiroot_url``?